eZ/Publish/API/Repository/Values/Content/Relation.php

```diff
@@ -50,6 +50,13 @@
      */
     const FIELD = 8;
 
+    /**
+     * the relation type ASSET is set for a relation to asset in an attribute value.
```

Comment:
Nitpic, it could be something like this
```
* The relation type ASSET is set for a relation to asset in an asset relation attribute value.
*
* This is kernel wise the exact same as FIELD with exectroion of two things:
* - It's reserved for asset field type (and field types extending it)
* - It is for use by behavior on top of API to for instance prompt users if they like to delete related assets as well when deleting content (assuming not other content points to it)
* @var int
```

However that can be followup, as I'm not 100% sure on the wording here so I leave that up to you as you know this better.